packages/web/src/pages/chat-page/components/ChatMessageList.tsx

```diff
@@ -241,6 +248,10 @@ export const ChatMessageList = forwardRef<HTMLDivElement, ChatMessageListProps>(
             ))}
           {!chat?.messagesSummary || chat.messagesSummary.prev_count > 0 ? (
             <LoadingSpinner className={styles.spinner} />
+          ) : messageListHeight > (ref.current?.clientHeight ?? 0) ? (
```

Comment:
Let's comment this or pull this out into a descriptively named variable (`isScrollable`)